rzup/src/main.rs

```diff
@@ -116,7 +118,24 @@ async fn main() {
     }
 }
 
+fn init() -> Result<()> {
+    // this is a rust revamp of rzup. Before this version, rzup was a bash
+    // script. In order to facilitate compatibility within our tools, we touch a
+    // file in a known location to indicate to the client that the rust
+    // impelemtnation of rzup is being utilized.
+    let new_rzup_indicator = utils::rzup_home()?.join("new-rzup");
```

Comment:
Can we depend on `risc-build` so that we don't have to duplicate this and have the possibility of it getting out of sync?